### ISSUE TITLE
Add style to an annotation muiListItem in companion window

### ISSUE THREAD

**Issue by ydvelaitzky** [ProjectMirador/mirador#3300]
Our annotationJson's item looks like the following:

`
 var item = {
 id: canvasId + annotInd,
 'type': 'Annotation',
 motivation: 'commenting',
 target: [{
            "id": canvasId,
            "selector": {
                "type": "SvgSelector",
                "value": "<path xmlns=\"http://www.w3.org/2000/svg\" d=\"" + xyStr + "\" fill=\"none\" fill-rule=\"nonzero\" stroke=\"" + colorName + "\" stroke-width=\"2\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/>"
            }
        }, body: {
            type: 'TextualBody',
            value: "<p>annotation title</p> <p style='color:gray'>additional text...</p>"
        }
}]
`

As you can see the **body.value** has a html that contains 2 P tags.
The problem: mirador ignores the style of these tags, and in companion-window it places the HTML tag without it's style.
Can you show us how to be able to insert in the annotation-companion-window a html with  a style?
Thank you!

**Comment by mejackreed** [comment]
Mirador sanitizes the HTML per the IIIF specification. However this is configurable in the settings by changing this setting value: https://github.com/ProjectMirador/mirador/blob/master/src/config/settings.js#L234

Acceptable sanitation values are in this file: https://github.com/ProjectMirador/mirador/blob/master/src/lib/htmlRules.js

**Comment by ydvelaitzky** [comment]
Thank you! now it works , and the tags are displayed with styles.
Bur FYI, We did not need to change the setting value, we've only need to add to the mirador initialization - an empty  **annotations.htmlSanitizationRuleSet** property, as the following:
` annotations: {
            htmlSanitizationRuleSet: {
            /*no value is set!*/
            } 
        }`
and as a magic - the style of the  body.value of the annotations was added to the tags.
